ui/components/component-library/text-field-search/README.mdx

```diff
@@ -1,37 +1,47 @@
 import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
 
+import { TextFieldBase, TextField } from '..';
+
 import { TextFieldSearch } from './text-field-search';
 
 # TextFieldSearch
 
-The `TextFieldSearch` allows users to enter text to search. It wraps the `TextField` component that adds a search icon to the left of the input.
+The `TextFieldSearch` allows users to enter text to search
```

Comment:
Simplifying description because we are including the prop tables from the children components